TODO.md

```diff
@@ -1,77 +1,74 @@
-* Switch default change scale to Minor Change
-* Bug: Various play buttons aren't mutually exclusive anymore - John
-* Bug: Start tone after user interaction (currently broken in Safari and Chrome)
-* Lineage can skip at beginning with lots of beats; investigate unnecessary re-rendering or other ways avoid skips - John
-* Actions at the bottom (rate, mutate), play at the bottom? - John
-* Switch star rating to how much to change slider - Luke
-* Solo/Gain for each section (ian)
-* Bug: All tracks can be Soloed at same time for drums
-* Duplicate beat in lineage - John
-* Pick lineage playback up where it left off, allow selecting starting beat when it's not currently playing
-* Load lineage from local storage
-* Name lineage
-* Rename beats
-* Undo 1 (or more) beat deletes
-* Additional ocave
-* Fix duplicate samplers
-* Add custom sections
-* URL sharing
+- Switch default change scale to Minor Change
```

Comment:
ha was the asterisk producing errors? Not sure why @johnthedebs even uses these.

Reply:
@ianboru @lukegrecki Fuck both ya'll, asterisks way better